scripts/helm/bitops.schema.yaml

```diff
@@ -25,12 +27,16 @@ helm:
         atomic:
           type: boolean
           parameter: atomic
+          dash_type: "-"
         force:
           type: boolean
           parameter: force
         dry-run:
           type: boolean
           parameter: dry-run
+        backend-config:
```

Comment:
`backend-config` is for terraform, not helm

Reply:
it also needs `dash_type: "-"`

Reply:
Helm was used for testing, I will switch this over to be in the Terraform schema.